upgrade-api-analyzer-sl.md

```diff
@@ -0,0 +1,95 @@
+---
+title: Upgrade API Analyzer tool
+page_title: How to Use the Upgrade API Analyzer
+description: How to use the Upgrade API Analyzer 
+slug: upgrade-api-analyzer-sl
+tags: api, analyzer, upgrade
+published: True
+position: 20
+---
+
+# How to Use the Upgrade API Analyzer 
+
+With every release of UI for Silverlight we’re trying to avoid introducing changes that affect the public API. However, sometimes such changes are needed for the product to evolve. We understand that such changes could cause issues when upgrading your app to the latest release. Mainly, the purpose of the __Telerik Upgrade API Analyzer__ is to make the upgrade process easier by comparing both versions and determining the problematic areas.
```

Comment:
малко ми се повтаря such changes във второто и третото изрчение :) може ли да го направиме нещо като ".. such changes are need for the product to evolve. We understand that this could cause issues when upgrading...."

"Mainly, the purpose of the.." на мен по-добре ми звучи "The main purpose of the..."